src/search/indexing/mod.rs

```diff
@@ -53,9 +58,14 @@ pub async fn index_mods(db: mongodb::Client) -> Result<(), IndexingError> {
     relevance_index
         .set_settings(&default_settings().with_ranking_rules(relevance_rules.into()))
         .map_err(IndexingError::IndexDBError)?;
-    relevance_index
-        .add_documents(docs_to_add.clone(), Some("mod_id"))
-        .map_err(IndexingError::IndexDBError)?;
+
+    for chunk in docs_to_add.chunks(MEILISEARCH_CHUNK_SIZE) {
+        // TODO: get meilisearch sdk to not require cloning (ie take a reference to docs_to_add)
+        // This may require making our own fork of it.
+        relevance_index
+            .add_documents(Vec::from(chunk), Some("mod_id"))
+            .map_err(IndexingError::IndexDBError)?;
```

Comment:
We can't make our own `Error` impl without making a new wrapper type, but we could implement `From<meilisearch_sdk::errors::Error>` for `IndexingError`.  We could also just wait until `meilisearch_sdk` updates to 0.2, as https://github.com/Mubelotix/meilisearch-sdk/issues/7 has been fixed, since then we can use `thiserror`'s `#[from]` attribute.

Reply:
Sounds good. merging